### PR TITLE
Nuke sel.go_to

### PR DESCRIPTION
**Pull request by lehinevych**
{{pytest: cfme/tests/configure/test_log_depot_operation.py cfme/tests/configure/test_server_roles.py  cfme/tests/configure/test_workers.py  cfme/tests/control/test_import_policies.py cfme/tests/infrastructure/test_provisioning_dialog.py -v --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit e2cddd664ff41e1ba5e868347de113a095c1a733:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cc14c0d779b795179c01227bba18a4571ed07ceb:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit bb54c41ca7c1bed0848967572ff916e1a6cfde62:

`cfme/fixtures/pytest_selenium.py`:
- :red_circle: Line 36:1: **F401** *'pytest' imported but unused*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1a20fa339526c7070798ac40bb209d4b768095ef:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/intelligence/reports/reports.py [hunk 1]
```diff
@@ -349,11 +350,11 @@ def data(self):
...
     def download(self, extension):
         self.navigate()
-        toolbar.select("Download")
-        sel.click(
-            "//table[contains(@class, 'buttons_cont')]"
-            "//img[contains(@class, 'btn_sel_img') and contains(@src, '{}.png')]".format(extension)
-        )
+        extentions_mapping = {'txt': 'Text', 'csv': 'CSV', 'pdf': 'PDF'}
+        try:
+            download_btn("Download as {}".format(extentions_mapping[extension]))
```
Comment (mfalesni) [inline cfme/intelligence/reports/reports.py]: s/extentions/extensions/

File: cfme/tests/intelligence/test_download_report.py [hunk 2]
```diff
@@ -43,24 +41,13 @@ def report():
     return reports.CannedSavedReport(path, reports.queue_canned_report(*path))
 
 
-@pytest.skip  # To be removed when we have solved the Docker issue
```
Comment (mfalesni) [inline cfme/tests/intelligence/test_download_report.py]: @psav Do we have the files download solved, since the browser runs in a separate container?

Comment (lehinevych) [inline cfme/tests/intelligence/test_download_report.py]: It looks like the answer is no, at least test fails there.

**Comment by psav** [comment]
As per my comment about moving to a private function, I didn't know the go_to we were talking about was a fixture. Am happy to delete.

File: cfme/intelligence/reports/reports.py (hunk 1 above)
Comment (psav) [inline cfme/intelligence/reports/reports.py]: Yeh lets fix this then we can merge :)

**Comment by dajoRH** [comment]
Lint report for commit f2ad8f1bc2d43b7b9dce0334b41d6715aa0c0e83:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by psav** [comment]
@mfalesni I'd like to get this merged, it looks good to me

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by psav** [comment]
Can we get this rebased and @mfalesni are you happy to merge this....I'd like it to get in

**Comment by lehinevych** [comment]
@psav @mfalesni rebased.

**Comment by dajoRH** [comment]
Lint report for commit 36827ada1250ae24b5a4667d1c68ecb02bb36c38:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3425bfa4e166f4a68202a487cadd2f7c4a43520e:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**